tests/Directory.Packages.props

```diff
@@ -1,24 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
-  <PropertyGroup>
-    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-    <NoWarn>NU1507</NoWarn>
-  </PropertyGroup>
+  <!-- Enable multi-level merging -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Packages.props, $(MSBuildThisFileDirectory)..))" />
```

Comment:
This ensures the global packages are also used for the tests, since they are not added in `Directory.Build.props` anymore...